src/sentry/static/sentry/app/components/commitRow.jsx

```diff
@@ -25,25 +28,23 @@ export default class CommitRow extends React.Component {
   render() {
     let {id, dateCreated, message, author, repository} = this.props.commit;
     return (
-      <li className="list-group-item" key={id}>
-        <div className="row row-center-vertically">
-          <div className="col-xs-10 list-group-avatar">
-            <Avatar size={36} user={author} />
-            <h5 className="truncate">{this.renderMessage(message)}</h5>
-            <p>
-              {tct('[author] committed [timeago]', {
-                author: (
-                  <strong>{idx(author, _ => _.name) || t('Unknown author')}</strong>
-                ),
-                timeago: <TimeSince date={dateCreated} />,
-              })}
-            </p>
-          </div>
-          <div className="col-xs-2 hidden-xs align-right">
-            <CommitLink commitId={id} repository={repository} />
+      <PanelItem key={id} align="center">
+        <Flex mr={2}>
+          <Avatar size={36} user={author} />
+        </Flex>
+        <Flex style={{flexGrow: 1, fontSize: 15}} direction="column">
+          <strong className="truncate">{this.renderMessage(message)}</strong>
```

Comment:
I think this would be nicer as a styled component that extends `TextOverflow`:

```
styled(TextOverflow)`
font-size: 15px;
font-weight: bold;
`;
```